workflows/qc/quality-check-standard.wdl

```diff
@@ -101,12 +97,16 @@ workflow quality_check {
                 flagstat_file=samtools_flagstat.outfile,
                 fastqc_files=fastqc.out_files,
                 qualimap_bamqc=qualimap_bamqc.results,
-                fastq_screen=fastq_screen.out_files,
                 qualimap_rnaseq=qualimap_rnaseq.results,
                 max_retries=max_retries
         }
     }
     if (experiment == "WGS" || experiment == "WES") {
+        File fastq_screen_db_defined = select_first([fastq_screen_db, "No DB"])
```

Comment:
I'm a bit unsure about this: I feel like there should be a better way to coerce an optional input into a required input but after looking through the GitHub Issues, most of the discussion is aimed at development spec and I don't know if 1.0 has a way to do this.

Reply:
I don't like it, but it's the best 1.0 way I could find.

Unfortunately the expression gets evaluated regardless of the evaluation of the if clause, which is why I had to include the `"no GTF/DB"` strings. Otherwise it was throwing errors that it couldn't select nothing, even though the expression logically shouldn't have been evaluated.

Reply:
To my knowledge, this is the best current way to handle this.